WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -4780,6 +4813,9 @@ Please install the %3$@ to use the app with this site.";
 /* A short label.  A call to action to load more posts. */
 "Load more posts" = "Load more posts";
 
+/* No comment provided by engineer. */
+"Loading" = "Loading";
```

Comment:
This string without reverse-DNS key [comes from Gutenberg](https://github.com/wordpress-mobile/gutenberg-mobile/blob/9a181e9abffa7c8d51305625d2614b85ad1a9f86/bundle/ios/GutenbergNativeTranslations.swift#L198).